setup.py

```diff
@@ -88,6 +90,7 @@ def _collect_requirements(requires):
     "plotly.PlotlyDataSet": [PANDAS, "plotly>=4.8.0, <6.0"],
     "plotly.JSONDataSet": ["plotly>=4.8.0, <6.0"],
 }
+redis_require = {"redis.PickleDataSet": ["redis~=4.1.0"]}
```

Comment:
Can we do this?
```suggestion
redis_require = {"redis.PickleDataSet": ["redis~=4.1"]}
```
For stable package we try to pin on major release, rather than minor.